packages/sc-deployer/src/index.ts

```diff
@@ -90,56 +139,64 @@ async function deploySC(
     account,
   );
 
-  await checkBalance(client, account);
+  // check deployer account balance
+  const coinsRequired = contracts.reduce((acc, contract) => acc + contract.coins.rawValue().toNumber(), 0);
```

Comment:
you should not use number here. just bignumbers

Reply:
or "Massacoins"

Reply:
good idea, fixed